src/main/java/application/rest/CustomerService.java

```diff
@@ -113,6 +114,7 @@
     @Metered(name="CustomerMeter",
             displayName="Customer Call Frequency",
             description="Rate of the calls made to CloudantDB")
+    @Traced(value = true)
```

Comment:
Do you want to include a name for this trace? It will be nice to have a name that identifies these traces.